### PR TITLE
updates to token and contribution docs

### PR DESCRIPTION
**Pull request by tuckernickman**
Updates to token launch plan, lunar cycles, reserve rate, and progressive decentralized governance, including information about roles.

### CODE REVIEW

**Comment by dissentertainer** [comment]
@pmoncada Since this is copy, can you review it and merge the PR? It might be good to have certain people who control copy PRs and others who control code PRs.

File: docs/token.md [hunk 1]
```diff
@@ -5,6 +5,19 @@ sidebar_label: $MOONEY token
...
 [Buy $MOONEY on Juicebox](https://juicebox.money/#/p/moondao)
+## Distribution & Tokenomics
+Governance Token MOONEY to be distributed in Lunar phases, with incremental decreases in the reserve rate over time. 
+
+Our funding goal: The primary goal is to decentralize the governance of MoonDAO, but in order to fund our project we need capital. $450,000 gives us enough to purchase at least one ticket, but tickets in auction have gone up to $27 Million dollars. This token raise won’t be our only opportunity to raise funds, but ideally this token gives us enough runway to achieve Moon phase 2
+## Reseve Rate Fluctuation
+
+**Lunar Cycle 0** - launch date **12/17/21 17:00 EST**: reserve rate of 500,000 MOONEY for every 1 ETH donated 
+
+**Lunar Cycle 1** -  **1/18/22 Time TBD**:
```
Comment (pmoncada) [inline docs/token.md:21]: Yeah we should probably have this convo soon, I could also see arguments to not raise it, I'm curious what people think in general!

**Review by pmoncada — APPROVED** [review]

**Comment by pmoncada** [comment]
Yep Max I agree, I can look at copy changes from now on!

File: docs/token.md (hunk 1 above)
Comment (tuckernickman) [reply docs/token.md:21]: It depends what people decide is best for the future of MoonDAO, we should bring this up with the community soon!

**Pull request merged**